### PR TITLE
Phase2-hgx138 Update HGCal v9 geometry to define fiducial boundaries correctly

### PR DESCRIPTION
**Pull request by bsunanda**
Proper utilization of mixed layer and non-mixed layer for HE-Silicon part

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

File: Geometry/HGCalCommonData/src/HGCalParametersFromDD.cc [hunk 1]
```diff
@@ -12,7 +12,7 @@
 #include <iostream>
 #include <iomanip>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```
Comment (ianna) [inline Geometry/HGCalCommonData/src/HGCalParametersFromDD.cc]: @bsunanda - please, comment it out.

**Review by ianna — CHANGES_REQUESTED** [review]

**Comment by bsunanda** [comment]
@ianna @kpedro88 @fabiocos It will be nice if this PR is integrated fast. The next sets of PR's including some earlier ones depend on this PR.

**Comment by ianna** [comment]
+1

**Comment by kpedro88** [comment]
+upgrade

**Comment by fabiocos** [comment]
+1

**Pull request merged**